auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java

```diff
@@ -33,16 +32,13 @@
     private final Context mContext;
     private final FlowParameters mFlowParameters;
     private final int mButtonText;
-    private final ForegroundColorSpan mLinkSpan;
 
     private SpannableStringBuilder mBuilder;
 
     private PreambleHandler(Context context, FlowParameters parameters, @StringRes int buttonText) {
         mContext = context;
         mFlowParameters = parameters;
         mButtonText = buttonText;
-        mLinkSpan = new ForegroundColorSpan(ContextCompat.getColor(mContext,
```

Comment:
What happened to this span?

Reply:
To my understanding - this span was to get the correct color on the URL links. This seem to be handled by `URLSpan` on it's own - given that `<item name="android:textColorLink">@color/fui_linkColor</item>` is set in the theme.

Reply:
👍 yep!